helm/tests/gateway/configmap_user_test.yaml

```diff
@@ -0,0 +1,53 @@
+suite: Test Gateway configmap section user
+templates:
+  - "gateway/gateway-configmap.yaml"
+tests:
+  - it: Test user config block
+    template: gateway/gateway-configmap.yaml
+    set:
+      userManagement:
+        email: test@test.com
+        password: password
+        name: name
+        username: luk
+        registration: 1
+        resetPassword: 2
+        blockedAccount: 3
+        mfaChallenge: 4
+        mfaVerifyAttempt: 5
+        webAuthn: 6
```

Comment:
**TODO**: is it possible to test at lest one complexe entry like it should be in a valid config?
ex:
```
  email:
    policy:
      pattern: ^[a-zA-Z0-9_+-]+(?:\.[a-zA-Z0-9_+-]+)*@(?:[a-zA-Z0-9-]+\.)+[a-zA-Z]{2,15}$
```

Reply:
added